src/iexamine.cpp

```diff
@@ -4904,6 +4907,27 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             break;
         }
 
+        case RAD_AWAY: {
+            patient.moves -= 500;
+            patient.add_msg_player_or_npc( m_info,
+                                           _( "The autodoc scanned you and detected a radiation level of %d mSv (%d mSv from items)." ),
+                                           _( "The autodoc scanned <npcname> and detected a radiation level of %d mSv (%d mSv from items)." ),
+                                           patient.get_rad(), patient.leak_level( "RADIOACTIVE" ) );
+            if( patient.get_rad() ) {
+                if( patient.has_effect( effect_pblue ) ) {
+                    patient.add_msg_player_or_npc( m_info,
+                                                   _( "The autodoc detected an anti-radiation drug in your bloodstream, so it decided not to administer another dose right now." ),
+                                                   _( "The autodoc detected an anti-radiation drug in <npcname>'s bloodstream, so it decided not to administer another dose right now." ) );
+                } else {
+                    add_msg( m_good,
```

Comment:
Shouldn't this be `add_msg_player_or_npc` as well?

Reply:
The text is identical for both player and NPC, so I thought generic `add_msg` will suffice.